drivers/radio/wcn14a2a.h

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright (c) 2018 Open Source Foundries Ltd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/types.h>
+#include <zephyr.h>
+
+#include <net/net_context.h>
+#include <net/net_if.h>
+
+#define MDM_BOOT_MODE_SEL_PIN		17
```

Comment:
I'd say this should go into its own file, and ideally, living somewhere on the k64f_frdm side.